src/main/java/ch/njol/skript/util/BlockStateBlock.java

```diff
@@ -407,4 +408,9 @@ public boolean isPassable() {
 	public BoundingBox getBoundingBox() {
 		return state.getBlock().getBoundingBox();
 	}
+	
+	@Override
+	public BlockSoundGroup getSoundGroup() {
+		return state.getBlock().getSoundGroup();
```

Comment:
Did you make sure this doesn't affect how this class works on Spigot?

Reply:
Im not entirely sure how to check this.

Reply:
```
on block can build check:
  broadcast "%event-block%"
```
If this code works normally on Spigot, then there should be no issues.

Reply:
Okay I ran this on Spigot, works perfectly fine 👍